### ISSUE TITLE
enableAccessLog filter actively disable routes that are not matched

### ISSUE THREAD

**Issue by rgritti** [zalando/skipper#1514]
**Describe the bug**
Filter `enableAccessLog` with parameter disables access logs of status codes that are not matched.

**To Reproduce**
1. set `skipper.Options.AccessLogDisabled = false`
2. add a route with `-> enableAccessLog(4, 5)`
3. make a requests that returns 2xx

**Expected behavior**
I expect to see the access logs

**Observed behavior**
Access logs for the request are not shown

**Comment by AlexanderYastrebov** [comment]
`enableAccessLog` [overrides](https://github.com/zalando/skipper/blob/master/docs/reference/filters.md#enableaccesslog) global Skipper `AccessLogDisabled` setting per route

**Comment by rgritti** [comment]
Right, but in my example above I have never explicitly disabled anything, but I end up with access logs for status codes 2xx disabled.

**Comment by szuecs** [comment]
@rgritti as far as I understand you set the default to log, but then you filter logging for the route to log only 4xx and 5xx.
So the behavior seems as you configured, not?

**Comment by rgritti** [comment]
@szuecs I think that if you **enable** the access logs globally, and then you **enable** specifically the 4xx and 5xx, you should see the 2xx as well, no?

**Comment by szuecs** [comment]
No, I think you should specify what you want to see, no inheritance!
